models/user.go

```diff
@@ -18,7 +18,8 @@ type User struct {
 	UpdatedAt time.Time  `json:"updated_at"`
 	DeletedAt *time.Time `json:"-"`
 
-	OrderCount int64 `json:"order_count,ommitempty" gorm:"-"`
+	OrderCount  int64      `json:"order_count,ommitempty" gorm:"-"`
```

Comment:
s/ommitempty/omitemtpy/

Reply:
Probably we want to have `order_count: 0` explicitly if there are no orders, right? Any frontend will usually prefer this.

So should we just drop the `omitempty`? This would also not result in an API change.

Reply:
I'm fine either way :)